actor/actionerror/shared_service_instance_not_found.go

```diff
@@ -0,0 +1,9 @@
+package actionerror
+
+// SharedServiceInstanceNotFound is returned when a service instance is not found when performing a share service.
+type SharedServiceInstanceNotFound struct {
```

Comment:
Please create a translatable error for this actionerror so that the error message can be localized. You can use this translatable error as an example: https://github.com/cloudfoundry/cli/blob/ee9d4748db79bc07352776e250383873c3e578cf/command/translatableerror/authorization_endpoint_not_found_error.go#L1-L12

Please also make sure the actionerror is converted to this new translatable error by adding it to this file: https://github.com/cloudfoundry/cli/blob/ee9d4748db79bc07352776e250383873c3e578cf/command/translatableerror/convert_to_translatable_error.go#L17

And update unit tests here: https://github.com/cloudfoundry/cli/blob/ee9d4748db79bc07352776e250383873c3e578cf/command/translatableerror/convert_to_translatable_error_test.go#L20
and here: https://github.com/cloudfoundry/cli/blob/ee9d4748db79bc07352776e250383873c3e578cf/command/translatableerror/translate_test.go#L32

Reply:
Added. That's some pro-level githubbing by the way 😄 (and a super helpful walkthrough!)